src/library.html

```diff
@@ -1,4 +1,4 @@
-<div id="mediaLibraryPage" data-role="page" class="page type-interior mediaLibraryPage librarySectionPage withTabs fullWidthContent" data-require="scripts/medialibrarypage">
+<div id="mediaLibraryPage" data-role="page" class="page type-interior mediaLibraryPage librarySectionPage withTabs fullWidthContent">
```

Comment:
Lots of plugins use `data-require` so you can't remove that from the code itself until they get updated. You can remove it from the pages here though.

Reply:
I believe most of those requires can be removed in the plugins repo. Most of them are the ones that begin with emby-*. Most of these are already required when loading the routes.js module. We need to look case by case to remove those requires, but should work as expected.